### PR TITLE
Fixes and enhancements for monitoring-user-defined-projects role

### PR DESCRIPTION
**Pull request by AdityaHonkalas**

### CODE REVIEW

File: playbooks/roles/monitoring-user-defined-projects/tasks/main.yml [hunk 1]
```diff
@@ -2,97 +2,13 @@
...
-    
-- name: Check the ServiceMonitor resource
-  shell: "oc -n {{ user_defined_namespace }} get servicemonitor --no-headers | wc -l"
-  register: service_monitor_count
-  until: service_monitor_count.stdout|int > 0
-  retries: 5
-  delay: 30
-
-- name: Create and deploy cluster-monitering-config ConfigMap instance
+- name: Create cluster-monitering-config ConfigMap for enabling thne user-workload-monitoring
```
Comment (yussufsh) [inline playbooks/roles/monitoring-user-defined-projects/tasks/main.yml:11]: nit:
```suggestion
- name: Create cluster-monitering-config ConfigMap for enabling the user-workload-monitoring
```

Comment (AdityaHonkalas) [reply playbooks/roles/monitoring-user-defined-projects/tasks/main.yml]: Done.

**Comment by AdityaHonkalas** [comment]
Hi @yussufsh, @pravin-dsilva 

Request you to review the final changes. Thanks !

**Review by yussufsh — APPROVED** [review]
/lgtm

**Pull request merged**